### DISCUSSION TITLE
Combobox dropdown menu width

### DISCUSSION THREAD

**Discussion by RaKiing45** [TomSchimansky/CustomTkinter#1825] [Q&A]
Is there a way to match dropdown menu width with combobox entry field?
![obraz](https://github.com/TomSchimansky/CustomTkinter/assets/124391524/a1bef86b-8fda-49e1-affa-10fe84cd4ea3)

**Comment by Akascape** [comment]
@RaKiing45 
Here is a custom dropdown menu you can try: https://github.com/Akascape/CTkScrollableDropdown
![screenshots](https://user-images.githubusercontent.com/89206401/236677843-8d8b76fd-6145-47b1-8f4d-b6a64b08e1ea.png)

**Reply by RaKiing45** [reply]
Hi, 
thanks for help! One thing I noticed is that the dropdown menu width is too long. Is there a fix for it?
![obraz](https://github.com/TomSchimansky/CustomTkinter/assets/124391524/90138296-bb0a-4be5-b593-eceb836488da)

**Reply by Akascape** [reply]
@RaKiing45 Not facing this issue on my device, don't know what's wrong.

**Comment by jklouz** [comment]
its possible to make dropdown options to be static state when selected? like in customtkinter documentation web page
![ctk](https://github.com/TomSchimansky/CustomTkinter/assets/101420599/a20e30b6-1666-45a2-b66a-8007b92133ac)
